### PR TITLE
Upgraded Linter to 3.5.0, AirBnB to 11.1.0

### PR DESCRIPTION
**Pull request by jdetle**
## Before Merge!!

Check if you want to ignore the unused variable exceptions and the no-undef exceptions.

### CODE REVIEW

**Comment by jdetle** [comment]
This is super frustrating because I can't replicate the error on my machine.

File: .eslintrc [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "parser": "babel-eslint",
-  "extends": "airbnb",
+  "extends": "airbnb/legacy",
```
Comment (rgbkrk) [inline .eslintrc:3]: There's one way to do it! This must be new.

File: .eslintrc [hunk 2]
```diff
@@ -11,13 +11,27 @@
...
+       "react/jsx-filename-extension": "off",
+       "import/extensions": [0, { "js": "never", "jsx": "never"}],
+       "max-len": [
+            2,
+            100, {
+                "ignoreComments": true,
+                "ignoreTrailingComments": true,
+                "ignoreUrls": true,
+                "ignoreTemplateLiterals": true
+            }
```
Comment (rgbkrk) [inline .eslintrc:33]: Thanks for digging deep into these options.

**Review by rgbkrk — APPROVED** [review]

**Comment by rgbkrk** [comment]
It's ok if we go with undef, one of these days we'll want to remove some of our escape hatches and line up with upstream (if it makes sense).

**Pull request merged**